src/custom/pages/Claim/types.ts

```diff
@@ -7,3 +8,11 @@ export type ClaimCommonTypes = {
   tokenCurrencyAmount: CurrencyAmount<Token>
   handleChangeAccount: (e: SyntheticEvent<HTMLButtonElement>) => void
 }
+
+export type ParsedUserClaim = UserClaimData & {
+  parsedAmount: string
```

Comment:
why "parsed", it looks like its an amount in string, so its not parsed

Reply:
why not `CurrencyAmount<Token>`?

Reply:
My idea for parsed was that is is ready for the UI. 
Changed to `currencyAmount: CurrencyAmount<Token> | undefined`